men_team_1/app/controllers/pairings_controller.rb

```diff
@@ -5,21 +5,21 @@ class PairingsController < ApplicationController
         # @mentee_of_user = @user.mentor_pairings
 
     def index
-        @mentors = []
+        @list_of_mentor_pairings = []
         @user.mentee_pairings.each do |pairing|
             if pairing.status == 'active'
-            @mentors << User.find(mentor.mentor_id)
+            @list_of_mentor_pairings << pairing
             end
         end
-        @mentors
-
-        @mentees = []
-        @user.mentor_pairings.each do |mentee|
-            @mentees << User.find(mentee.mentee_id)
+        @list_of_mentor_pairings
```

Comment:
You could define this much more compactly:

`@list_if_mentor_pairings = @user.mentee_pairings.where(status: "active")`
You also don't need to return `@list_of_mentor_pairings` here. The index controller action is just defining the instance variables for use in view. 

@smohan01 mentioned you were getting an error in dev here. I would test in console and make sure nothing is returning nil. The error message should also give you a line number and file where something is wrong. As per our meeting on Friday, make sure that @user is available here. You can also insert `byebug`  after line 7 and try loading the page. This will pause the load and turn your server output into a usable console, where you can check the different variables and data.